test/e2e/lib/helpers/broker_event_transformation_test_with_source.go

```diff
@@ -103,6 +104,58 @@ func BrokerEventTransformationTestHelper(client *lib.Client, brokerURL url.URL,
 	}
 }
 
+func BrokerEventTransformationTracingTestHelper(client *lib.Client, projectID string, brokerURL url.URL, brokerName string) {
+	client.T.Helper()
+	senderName := helpers.AppendRandomString("sender")
+	targetName := helpers.AppendRandomString("target")
+
+	// Create a target Job to receive the events.
+	makeTargetJobOrDie(client, targetName)
+
+	// Create the Knative Service.
+	kserviceName := CreateKService(client, "receiver")
+
+	// Create a Trigger with the Knative Service subscriber.
+	triggerFilter := eventingtestresources.WithAttributesTriggerFilter(
+		eventingv1alpha1.TriggerAnyFilter, eventingv1alpha1.TriggerAnyFilter,
```

Comment:
change this and other places to eventingv1beta1. 
v1alpha1 was removed and it will fail. Trying to update deps here: https://github.com/google/knative-gcp/pull/1222